server/web/edit/index.hbs

```diff
@@ -50,6 +50,8 @@ This edit will create a new revision.
       <label for="title">Title <em title="This field is required">*</em> </label>
       <input type="text" name="title" id="title" value="{{page.title}}" required>
       {{error page.titleError}}
+
+      <input type="hidden" name="slug" id="slug" value="{{page.slug}}">
```

Comment:
since we're sharing the schema for creating and editing, need to include page slug in payload